resources/lang/es.json

```diff
@@ -0,0 +1,507 @@
+{
+	"14 days": "14 días",
+	"A short description, as to what is being billed": "Una breve descripción de lo que se está facturando.",
+	"Absence overview": "Resumen de ausencias",
+	"Absence registered": "Ausencia registrada",
+	"Access": "Acceso",
+	"Access Denied": "Acceso Denegado",
+	"Access not given, try again": "No se ha concedido el acceso, prueba de nuevo",
+	"Action": "Acción",
+	"Activity": "Actividad",
+	"Add Comment": "Añadir comentario",
+	"Add Note": "Añadir nota",
+	"Add new Role": "Añadir nuevo Rol",
+	"Add time": "Añadir tiempo",
+	"Additional comments": "Comentarios adicionales",
+	"Address": "Dirección",
+	"All Clients": "Todos los Clientes",
+	"All Departments": "Todos los Departamentos",
+	"All Documents": "Todos los Documentos",
+	"All Files": "Todos los Archivos",
+	"All Leads": "Todos los Leads",
+	"All Projects": "Todos los Proyectos",
+	"All Roles": "Todos los Roles",
+	"All Tasks": "Todas las Tareas",
+	"All Unqualified Leads": "Todos los Leads no calificados",
+	"All Users": "Todos los Usuarios",
+	"All clients": "Todos los clientes",
+	"All tasks": "Todas las tareas",
+	"All users": "Todos los usuarios",
+	"All your active clients will be shown here": "Todos tus clientes activos se mostrarán aquí",
+	"Allowed": "Permitido",
+	"Amount": "Cantidad",
+	"Amount due": "Cantidad adeudada",
+	"Amount in": "Cantidad en",
+	"Amount of users": "Número de usuarios",
+	"Api key": "Clave de Api",
+	"Appointments": "Citas",
+	"Are you sure you want to close the lead?": "¿Estás seguro de que quieres cerrar el lead?",
+	"Are you sure you want to convert the lead to an order?": "¿Estás seguro de que quieres convertir el lead a un pedido?",
+	"Are you sure you want to convert the lead to qualified?": "¿Estás seguro de que quieres convertir el lead en calificado?",
+	"Are you sure?": "¿Estás seguro?",
+	"Assign Client": "Asignar cliente",
+	"Assign client": "Asignar cliente",
+	"Assign department": "Asignar departamento",
+	"Assign new user": "Asignar nuevo usuario",
+	"Assign role": "Asignar rol",
+	"Assign user": "Asignar usuario",
+	"Assigned": "Asignado",
+	"Assigned to": "Asignado a",
+	"Assigned user": "Usuario asignado",
+	"Attach invoice as PDF": "Adjuntar factura como PDF",
+	"Billing": "Facturación",
+	"Billing integrations": "Integraciones de facturación",
+	"Billy products": "Productos Billy",

```

Comment:
This could be a typo, but not really sure about "Productos Billy", maybe "Productos Facturables" what would make a little more sense.

Reply:
Does Billy mean billable? I thought it might be a name of a external resource 🤔

Going to fix this and spaces right now 😉

Reply:
I think I got the Billy part, It is an accountant software in denmark: [](https://www.billy.dk/) I found it because it sounded weird to me that Billy was also a word in Danish :)

May be we got this?

Reply:
I see, nice work man!